app/services/promo/registrations_stats_fetcher.rb

```diff
@@ -62,6 +64,10 @@ def api_authorization_header
   end
 
   def query_string(referral_codes)
-    "?" + referral_codes.map { |referral_code| "referral_code=#{referral_code}"}.join("&")
+    query_string = "?"
```

Comment:
Preferably we use the older version since it's much more readable.

Reply:
Sounds good, fixed.